packages/theme/src/cli/utilities/theme-environment/storefront-renderer.test.ts

```diff
@@ -16,17 +13,16 @@ vi.mock('@shopify/cli-kit/node/http', async () => {
 })
 
 const successResponse = {ok: true, status: 200, headers: {get: vi.fn()}} as any
-const sessionCookies = {
-  storefront_digest: '00001111222233334444',
-  _shopify_essential: ':00112233445566778899:',
-}
 
-const session = {
-  token: 'token_abc123',
+const session: DevServerSession = {
+  token: 'admin_token_abc123',
```

Comment:
🤔 I wonder if it's worth renaming `token` to `adminToken`, though that would mean that `AdminSession` parent type would need to be renamed

I think we can defer this for now but if we reach a state of >=3 tokens I think we should consider renaming `token`

Reply:
The session is indeed handling a lot of tokens. Another possible direction could be to introduce a getter in the `DevServerSession`. For now, I'm keeping the session as it is, as this PR aims to only refresh the session object, however, that's certainly a valuable approach to consider.